spec/models/manageiq/providers/microsoft/infra_manager/refresher_spec.rb

```diff
@@ -257,19 +257,19 @@ def assert_specific_guest_devices
       :start_connected => true,
     )
 
-    expect(v2.hardware.guest_devices[0]).to have_attributes(
-      :device_name     => "en_visio_professional_2016_x005F_x86_x005F_x64_dvd_6962139",
+    expect(v2.hardware.guest_devices.sort.first).to have_attributes(
```

Comment:
maybe `.order(:device_name).first` ?  It's a small optimization so we don't bring back all of the guest devices I think